test/integration/disruption/disruption_test.go

```diff
@@ -104,7 +104,7 @@ func TestPDBWithScaleSubresource(t *testing.T) {
 
 	stopCh := make(chan struct{})
 	informers.Start(stopCh)
-	go pdbc.Run(stopCh)
+	go pdbc.Run(context.TODO())
```

Comment:
As this is a test that involves multiple goroutine creations, to avoid goroutine leaking, a best practice is to create a cancellable context in the beginning, and reuse all across this test:

```go
	ctx, cancel := context.WithCancel(context.Background())
	defer cancel()
	
	informers.Start(ctx.Done())
	go pdbc.Run(ctx)
	......
```

(applies to this file L240 and L355 and evictions_test.go)

Reply:
Same as above.

Reply:
This comment is not the same as the above one. It's regarding goroutine leaking.

Reply:
Yeah, this comment from Wei is reasonable, even though we're not falling, it might lead to undesirable outcomes in the CI, so it's best to address it now.

Reply:
I meant I did not think about this as this is within tests. But I am making changes.